You can import CSV data and use it in Python with the pandas library.

In [36]:
import pandas as pd

Import the CSV file from exactly where it is in your file system relative to your Jupyter Notebook file system. For me, my download went straight to my downloads folder as a folder and a zip file. In the read_csv command below, I navigate to the csv file, which, for me, is in a folder in my Downloads folder called "forest-area-as-share-of-land-area".

If you need help with this part, please let me know!

In [56]:
forest_area = pd.read_csv('Downloads/forest-area-as-share-of-land-area/forest-area-as-share-of-land-area.csv')

You can look at the data here. Pandas makes it really easy to see it because it gives it headers and whatnot.

In [38]:
forest_area

Entity Code  Year  Forest cover
0     Afghanistan  AFG  1990      1.850994
1     Afghanistan  AFG  1991      1.850994
2     Afghanistan  AFG  1992      1.850994
3     Afghanistan  AFG  1993      1.850994
4     Afghanistan  AFG  1994      1.850994
...           ...  ...   ...           ...
7969     Zimbabwe  ZWE  2016     45.570273
7970     Zimbabwe  ZWE  2017     45.451183
7971     Zimbabwe  ZWE  2018     45.332093
7972     Zimbabwe  ZWE  2019     45.213002
7973     Zimbabwe  ZWE  2020     45.093912

[7974 rows x 4 columns]

Here is an example of how to look at one country in this dataset.

In [39]:
forest_area[forest_area['Entity'] == 'Brazil']

Entity Code  Year  Forest cover
922  Brazil  BRA  1990     70.458021
923  Brazil  BRA  1991     70.005654
924  Brazil  BRA  1992     69.553288
925  Brazil  BRA  1993     69.100922
926  Brazil  BRA  1994     68.648556
927  Brazil  BRA  1995     68.196190
928  Brazil  BRA  1996     67.743823
929  Brazil  BRA  1997     67.291457
930  Brazil  BRA  1998     66.839091
931  Brazil  BRA  1999     66.386725
932  Brazil  BRA  2000     65.934359
933  Brazil  BRA  2001     65.461671
934  Brazil  BRA  2002     64.988983
935  Brazil  BRA  2003     64.516295
936  Brazil  BRA  2004     64.043608
937  Brazil  BRA  2005     63.570920
938  Brazil  BRA  2006     63.098232
939  Brazil  BRA  2007     62.625545
940  Brazil  BRA  2008     62.152857
941  Brazil  BRA  2009     61.680169
942  Brazil  BRA  2010     61.207482
943  Brazil  BRA  2011     61.023328
944  Brazil  BRA  2012     60.839175
945  Brazil  BRA  2013     60.655021
946  Brazil  BRA  2014     60.470868
947  Brazil  BRA  2015     60.286715
948  Brazil  BRA  2016     60.112867
949  Brazil  BRA  2017     59.939020
950  Brazil  BRA  2018     59.765173
951  Brazil  BRA  2019     59.591325
952  Brazil  BRA  2020     59.417478

There's a lot of information here. 

We can see that in 1990, 70% of Brazil's land cover was covered in forest. In 2020, 59% was.

With this information, we can start to calculate things like average forest loss per year, forest loss per decade, etc. Also, if we consider outside information, like political institutional changes, we can make hypotheses about how these changes affected forest land cover in Brazil.

And here is an example of how to look at the forest loss in Brazil between 2000 and 2010.

In [40]:
brazil = forest_area[forest_area['Entity'] == 'Brazil']

In [68]:
brazil_2010 = float(brazil[brazil['Year'] == 2010]['Forest cover'])

In [69]:
brazil_2000 = float(brazil[brazil['Year'] == 2000]['Forest cover'])

In [70]:
diff = brazil_2000 - brazil_2010

In [71]:
diff

4.726877032449799

So, Brazil, between 2000 and 2010, an additional 4.7% of Brazil's land cover was converted from forested land to non-forested land.

This is my new edit!!